packages/next/babel.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Babel preset to provide Gatsby support for Nx.
```

Comment:
should this say "Gatsby"? seeing how it's in the `next` directory, i assume this needs to say "Next"?

Reply:
Yes it should say Next.js